### PR TITLE
#1555 update mobile docs

### LINKED ISSUE #1555: Update mobile setup documentation
## Is your feature request related to a problem? Please describe.

Placeholder for updating and getting feedback on mobile setup documentation.

## Describe the solution you'd like

Updating mobile documentation based on feedback during Nepal trip.

## Describe alternatives you've considered

N/A.

## Additional context

Creating an issue to get feedback on PR from NZ before making changes.

### PR DESCRIPTION
**Pull request by wlthomson**
Fixes #1555.

## Change summary

Just some minor tidyups to `README.md` and a few additions to review in combination with some updates of the wiki as part of prep for upcoming mobile presentation. 

The update to the description/functionality is just me trying to make mobile sound cool to newcomers, not really relevant :). Main changes are to restructure to make the `README.md` more of a self-contained installation and development "quickstart", with the wiki covering broad stuff like configuration (mSupply setup etc.), design and code conventions, and learning resources.

Copy and paste [here](https://jbt.github.io/markdown-editor/) to get a better idea of what things look like.

Proposed wiki changes:

- Fix up home page (broken links).
- Update setup sidebar (out-of-date with current wiki page).
- Remove stuff duplicated in `README.md` (e.g. basic environment setup, Realm Browser).
- Remove test suite (don't think we use this?).
- Possibly update code design, but at this stage not a good use of time (eep).

### CODE REVIEW

**Comment by Chris-Petty** [comment]
I agree with the wiki tender loving care suggestions, make an issue and go ham

File: README.md [hunk 1]
```diff
@@ -1,55 +1,93 @@
...
 
-Instructions on the [setup page](https://github.com/sussol/mobile/wiki/Setup#setting-up-msupply-server)
+- Install Android Studio and SDK tools: https://developer.android.com/studio.
+- Install SDKMAN for managing Java versions: https://sdkman.io/.
 
-### Set Up for Development
+#### React Native
 
-Instructions on the [setup page](https://github.com/sussol/mobile/wiki/Setup)
+- Install nvm for managing Node versions: https://github.com/nvm-sh/nvm.
```
Comment (diegoevangelisti) [inline README.md]: As a suggestion, a new line can be added after this one, mentioning which is the node.js version that need to be install/selected after having nvm running properly (e.g. node v10.13.0). That can avoid future problems during the installation.

Comment (wlthomson) [reply README.md]: We have a `.nvm` file in the repo, so `nvm use .` should take care of that :).

**Comment by wlthomson** [comment]
@diegoevangelisti @Rebecca367 Going to go ahead and merge this, but if you have any problems/issues with mobile setup in the future, I think it would be good practice to open an issue asking for improved documentation. Feel free to mention me (and @Chris-Petty?) or anyone else you think might have a good understanding of the domain. That will help us keep our docs up-to-date and as useful as possible :).

**Comment by diegoevangelisti** [comment]
> @diegoevangelisti @Rebecca367 Going to go ahead and merge this, but if you have any problems/issues with mobile setup in the future, feel free to open an issue asking for improved documentation. You can mention me (and @Chris-Petty?) or anyone else you think might have a good understanding of the domain. That will help us keep our docs up-to-date and as useful as possible :).

Sure @wlthomson, will do. Totally agree that we need to keep on collaborating to have the docs updated for all of us and future users as well.

**Comment by Rebecca367** [comment]
Yes, I agree with you two. There were several issues happening to me when I
was installing softwares. I will look into the doc. to see what I was
missing and what more I can do to add on to the doc. Thanks for
mentioning this Will.

**Pull request merged**